index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>| Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="mobile-main">
+        <div class="mobile-header">
+            <div class="app-bar">
+                <div class="logo">
+                    My Logo
+                </div>
+                <div class="menu">
+                    <img src="/Union1.png" alt="">
+                </div>
+            </div>
+            <div class="header-main-content">
+                <div class="introduction-primary-text">
+                    I’m Andile
+                    Glad to see you!
+                </div>
```

Comment:
Glad to see you too 🤝 but how about we make the main title an `h1` tag? 🤔 
  * If you want that google (and other search engines) to know about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info. 🔗